name.abuchen.portfolio.tests/src/name/abuchen/portfolio/util/TradeCalendarTest.java

```diff
@@ -159,66 +158,97 @@ public void testMovingHolidays()
+        assertThat(calendar.isHoliday(LocalDate.parse("2001-09-14")), is(true)); // closed after terrorist attacks (9/11)
+        assertThat(calendar.isHoliday(LocalDate.parse("2001-09-17")), is(false)); // reopen market (Monday)
     }
 
     @Test
-    public void testLSE()
+    public void testTradeCalenderLSE()
     {
         // See https://www.londonstockexchange.com/trade/trading-access/business-days
 
         TradeCalendar calendar = TradeCalendarManager.getInstance("lse");
 
-        assertThat(calendar.isHoliday(LocalDate.parse("2019-01-01")), is(true));
+        // New Year
+        assertThat(calendar.isHoliday(LocalDate.parse("2021-01-01")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2022-01-01")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2023-01-01")), is(true));
+
+        // Good Friday
         assertThat(calendar.isHoliday(LocalDate.parse("2019-04-19")), is(true));
-        assertThat(calendar.isHoliday(LocalDate.parse("2019-04-22")), is(true));
-        assertThat(calendar.isHoliday(LocalDate.parse("2019-05-06")), is(true));
-        assertThat(calendar.isHoliday(LocalDate.parse("2019-05-27")), is(true));
-        assertThat(calendar.isHoliday(LocalDate.parse("2019-08-26")), is(true));
-        assertThat(calendar.isHoliday(LocalDate.parse("2019-12-25")), is(true));
-        assertThat(calendar.isHoliday(LocalDate.parse("2019-12-26")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2021-04-02")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2022-04-15")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2023-04-07")), is(true));
 
-        assertThat(calendar.isHoliday(LocalDate.parse("2021-12-24")), is(false));
-        assertThat(calendar.isHoliday(LocalDate.parse("2021-12-25")), is(true)); // Saturday and Christmas Day
-        assertThat(calendar.isHoliday(LocalDate.parse("2021-12-26")), is(true)); // Sunday and Boxing Day
-        assertThat(calendar.isHoliday(LocalDate.parse("2021-12-27")), is(true)); // Monday, substitute for holiday
-        assertThat(calendar.isHoliday(LocalDate.parse("2021-12-28")), is(true)); // Tuesday, substitute for holiday
-        assertThat(calendar.isHoliday(LocalDate.parse("2021-12-29")), is(false));
-        assertThat(calendar.isHoliday(LocalDate.parse("2021-12-30")), is(false));
-        assertThat(calendar.isHoliday(LocalDate.parse("2021-12-31")), is(false));
-        assertThat(calendar.isHoliday(LocalDate.parse("2022-01-01")), is(true)); // Saturday and New Year
-        assertThat(calendar.isHoliday(LocalDate.parse("2022-01-02")), is(true)); // Sunday
-        assertThat(calendar.isHoliday(LocalDate.parse("2022-01-03")), is(true)); // Monday, substitute for holiday
-        assertThat(calendar.isHoliday(LocalDate.parse("2022-01-04")), is(false));
+        // Eastern
```

Comment:
Should be “Easter Monday”. (The whole set of holidays is called Easter, not Eastern, which is a direction.)

Reply:
And before that there was nothing... :-/ 
I have made the effort to go calendrical after the holidays. That there is now Easter and not "Easter Monday" is now hopefully not so bad...